client/source/components/RelatedProducts/RelatedListCard.jsx

```diff
@@ -41,12 +45,13 @@ const RelatedListCard = function CreateRelatedListCard({ relatedProdId }) {
   };
 
   return (
-    <StyledRelatedListCard onClick={handleCardClick}>
+    <StyledRelatedListCard onClick={handleCardClick} ref={ref}>
       <img alt={productDetail.name} src={imageURL} />
       <HiOutlineStar className="compareButton" onClick={onOpen} />
-      <h3>{productDetail.category}</h3>
+      <h3 data-testid="card-category">{productDetail.category}</h3>
       <h4>{productDetail.name}</h4>
       <h5>${productDetail.default_price}</h5>
+      <AvgStarArray />
       <Modal isOpen={isOpen} onClose={onClose}>
         <CompareTable
```

Comment:
Currently when you click _inside_ the open modal window, it closese and changes the loaded hero garment (prodID).

Is this expected behavior?